src/definitions/modules/calendar.js

```diff
@@ -1276,6 +1276,8 @@ $.fn.calendar.settings = {
       if (text.length === 0) {
         return null;
       }
+      // Reverse date and month in some cases
+      text = settings.monthFirst ? text : text.replace(/([0-9]+)[\/\-]([0-9]+)/,'$2/$1');
```

Comment:
Lets also support a dot

```
text.replace(/([0-9]+)[\/\-\.]([0-9]+)/
```

, just because most of the people which are used to dd.mm.yyyy are most likely using a dot and won't enter 03-12-2019
Without the dot support it is still failing.

Reply:
Enhanced with by, 2359db3, thank you @lubber-de !